models/public/colorization-v2-norebal/model.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+    The `colorization-v2-norebal` model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
```

Comment:
Please don't use Markdown syntax here. This is supposed to be plain text.

Reply:
@IRDonch corrected

Reply:
You still have Markdown links.

Reply:
@IRDonch corrected.